### ISSUE TITLE
How to reset overlays on page?

### ISSUE THREAD

**Issue by jackhagley** [dhowe/AdNauseam#2423]
### ISSUE
I create an overlay, but it accidentally hides elements that I want

### WORKAROUND
turn off Ad Nauseam on page entirely
This is not ideal

### PROPOSED SOLUTION
Button in the dropdown menu "Reset overlays on this page" to return it to pre-blocked state

**Comment by mneunomne** [comment]
@jackhagley what exactly you mean by overlay? a cosmetic filter?

**Comment by jackhagley** [comment]
Sorry I mean I have used the right click context menu and blocked
element(s). But as a human and a user I have made mistakes and therefore I
would like an easy way to reset.
Possibly there is a way to do this by removing strings somewhere, but this
is not user friendly as some of the naming of these can be extremely opaque.

**Comment by mneunomne** [comment]
I'm not sure if there is a good way to implement this. Because sometimes you could have added a cosmetic rule that is not domain-specific. on the uBlock menu you can disable the cosmetic filters on the current page. But I think the best way to reverse this is to simply delete the filters that were added in the "My Filters" tab in the dashboard.

**Comment by dhowe** [comment]
Seems the simplest answer would be to 'use the uBlock menu and disable the cosmetic filters on the current page', yes ? 

If this is a solution, we could then discuss whether it makes sense to add a button...

**Comment by mneunomne** [comment]
I don't think it makes sense to add a button in this case. to have an UI button knowing there is a current filter in "my filters" matching the current domain string would require some unnecessary computing and de-compiling the 1p-filter just for 1 small feature.

**Comment by jackhagley** [comment]
I mean – it doesn't seem unnecessary to me, but as I won't be the one doing
the work to make it happen, I take your point

**Comment by mneunomne** [comment]
Sorry @jackhagley I didn't mean it's an unnecessary feature, but an additional computing of filters on every page-load can be costly.

**Comment by jackhagley** [comment]
No problem at all :)
